schema.sql

```diff
@@ -7,6 +7,18 @@ CREATE TABLE animals (
   escape_attempts INT,
   neutered BOOLEAN,
   weight_kg DECIMAL(5,2),
-  species VARCHAR(255),
+  species_id INT FOREIGN KEY REFERENCES species(id),
+  owner_id INT FOREIGN KEY REFERENCES owners(id),
   PRIMARY KEY(id)
 );
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INT
+);
+
+CREATE TABLE species ( 
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
```

Comment:
- [ ] [REQUIRED] Kindly note that there is a part of the project requirement that state that you should modify the `animals` table: Remove column `species`, Add column `species_id` which is a foreign key referencing `species` table, Add column `owner_id` which is a foreign key referencing the `owners` table.

You can archive this using the `ALTER TABLE` command. ALTER TABLE is a SQL command used to modify an existing database table by adding, removing, or modifying its columns, constraints, or indexes. 

Let's fix this 💪🏽

Reply:
Hello @Whoistolu, Thanks for your kind feedback. I did so in SQL but I didn't know that I have to write it in the file. Thanks for letting me know!

Reply:
You are welcome @Muhammad0602 👍🏽